classes/controller/menu/class-classic-blocks-usages.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * Blocks Usage class
+ *
+ * @package P4BKS\Controllers\Menu
+ * @since 1.40.0
+ */
+
+namespace P4GBKS\Controllers\Menu;
+
+/**
+ * Show posts using classic blocks.
+ */
+class Classic_Blocks_Usages extends Controller {
+	/**
+	 * Create menu/submenu entry.
+	 */
+	public function create_admin_menu() {
+
+		$current_user = wp_get_current_user();
+
+		if ( in_array( 'administrator', $current_user->roles, true ) && current_user_can( 'manage_options' ) ) {
+			add_submenu_page(
+				P4GBKS_PLUGIN_SLUG_NAME,
+				__( 'Classic block usages', 'planet4-blocks-backend' ),
```

Comment:
I would use `usage` (singular) because it is used as singular in `Blocks > Usage`